code/tier_classification/run_tier_classification.R

```diff
@@ -46,14 +49,29 @@ cosmic_resistance <- readr::read_tsv(file.path(data_dir, 'CosmicResistanceMutati
 
 # annotate tier classification
 if(nrow(oncokb_anno) > 0){
+  # annotate tier 
   all_findings_output <- tier_classification(all_findings_output = all_findings_output, 
```

Comment:
does this remove all old tier classification rules? 

just want to confirm as the goal is to layer on CIVIC as additional evidence criterion (E.g. search through oncokb, cosmic, hotspot + CIVIC to categorize in a tier).

Reply:
Yes - I am not replacing it, just adding it. Basically, I left the original tier annotation as is and then annotate based on CIVIC. And then the final tier annotation is the higher tier (e.g., CIVIC tier II, previous tier I, it would be tier I).